SQLAssignment.txt

```diff
@@ -0,0 +1,274 @@
+1) List all the columns of the Salespeople table.
+Ans-> select * from salespeople;
++------+---------+-----------+------+
+| SNUM | SNAME   | CITY      | COMM |
++------+---------+-----------+------+
+| 1001 | Peel    | London    |   12 |
+| 1002 | Serres  | Sanjose   |   13 |
+| 1003 | AxelRod | New York  |   10 |
+| 1004 | Motika  | London    |   11 |
+| 1007 | Rifkin  | Barcelona |   15 |
+| 1008 | Fran    | London    |   25 |
++------+---------+-----------+------+
+
+2)List all customers with a rating of 100.
+Ans->select * from customers where Rating='100';
++------+---------+--------+--------+------+
+| CNUM | CNAME   | CITY   | RATING | SNUM |
++------+---------+--------+--------+------+
+| 2001 | Hoffman | London |    100 | 1001 |
+| 2006 | Clemens | London |    100 | 1001 |
+| 2007 | Pereira | Rome   |    100 | 1004 |
++------+---------+--------+--------+------+
+4) Arrange the Order table by descending customer number.
+Ans->select * from orders order by CNUM desc;
++------+---------+------------+------+------+
+| ONUM | AMT     | ODATE      | CNUM | SNUM |
++------+---------+------------+------+------+
+| 3001 |   18.69 | 1990-10-03 | 2008 | 1007 |
+| 3006 | 1098.16 | 1990-10-03 | 2008 | 1007 |
+| 3002 |  1900.1 | 1990-10-03 | 2007 | 1004 |
+| 3008 |    4723 | 1990-10-05 | 2006 | 1001 |
+| 3011 | 9891.88 | 1990-10-06 | 2006 | 1001 |
+| 3007 |   75.75 | 1990-10-04 | 2004 | 1002 |
+| 3010 | 1309.95 | 1990-10-06 | 2004 | 1002 |
+| 3005 | 5160.45 | 1990-10-02 | 2003 | 1002 |
+| 3009 | 1713.23 | 1990-10-04 | 2002 | 1003 |
+| 3003 |  767.19 | 1990-10-03 | 2001 | 1001 |
++------+---------+------------+------+------+
+12) List the names and commissions of all salespeople in London.
+Ans-> select SNUM ,COMM from salespeople where city='london';
++------+------+
+| SNUM | COMM |
++------+------+
+| 1001 |   12 |
+| 1004 |   11 |
+| 1008 |   25 |
++------+------+
+11) Find all the customers in SanJose who have a rating above 200.
+Ans->select cname from customers where city= 'SanJose' and rating>='200';
++----------+
+| cname    |
++----------+
+| Liu      |
+| Cisneros |
++----------+
+5) Find which salespeople currently have orders in the order table.
+Ans->select Sname from salespeople INNER JOIN orders ON salespeople.snum=orders.snum;
++---------+
+| Sname   |
++---------+
+| Peel    |
+| Peel    |
+| Peel    |
+| Serres  |
+| Serres  |
+| Serres  |
+| AxelRod |
+| Motika  |
+| Rifkin  |
+| Rifkin  |
++---------+
+6) List names of all customers matched with the salespeople serving them.
+Ans->select cname from customers INNER JOIN salespeople ON customers.snum=salespeople.snum;
++----------+

```

Comment:
This is not a correct output. You should display the name of salespeople as well along with customers.